lib/config/config-file.js

```diff
@@ -210,6 +244,8 @@ function loadConfigFile(filePath) {
             default:
                 config = loadLegacyConfigFile(filePath);
         }
+    } else if (filePath.indexOf("eslint-plugin") >= 0) {
```

Comment:
The prefix is already stripped by `resolve()` before it sends the path to `loadConfigFile`.